src/components/ChooseCheckGroup/ChooseCheckGroup.tsx

```diff
@@ -32,12 +32,25 @@ export const ChooseCheckGroup = () => {
   );
 };
 
-const getStyles = (theme: GrafanaTheme2) => ({
-  container: css({
-    width: `100%`,
-    display: `grid`,
-    gridTemplateColumns: `repeat(auto-fit, minmax(200px, 400px))`,
-    gap: theme.spacing(2),
-    textAlign: `center`,
-  }),
-});
+const getStyles = (theme: GrafanaTheme2) => {
+  const twoColumnsQuery = `@media (max-width: ${theme.breakpoints.values.xxl}px)`;
+  const oneColumnQuery = `@media (max-width: ${theme.breakpoints.values.sm}px)`;
```

Comment:
We should add the container queries, too (grep search for `@container` in the repo and we have a few examples). That would solve for when users have the megamenu docked at certain resolutions.

![The Choose a check type at 1600px resolution. It has four cards in a row that look squashed and some content is being cut off the bottom of the first card](https://github.com/user-attachments/assets/d726bc83-e80d-4db3-b134-e6b5deb331c8)

Reply:
Great suggestion! I've added them.